.github/workflows/ci.yml

```diff
@@ -25,6 +25,10 @@ jobs:
           tools: composer:v2
           coverage: none
 
+      - name: Emulate PHP 8.3
+        run: composer config platform.php 8.3.999
```

Comment:
why do we need this? do we have dependencies that limit to php 8.3 and don't install with 8.4?

Reply:
phpspec disallows PHP 8.4